src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -1287,88 +1284,62 @@ public static ISurfacesPrx create(ImagePlus imp, int timepointOffset) throws Err
             return surface;
         }
 
-        /**
-         * create a new surfaces object from a Label ImagePlus
+/**
+         * Create a new surfaces object from this ImagePlus
          *
-         * @param impLabel the image to get a Surfaces from.
-         *                 A label image, each label will be a surface of the Surfaces object.
-         * @return the ISurfacesPrx with individual surface for each label value
+         * @param impLabel the label image to get a surface from. can be 8 or 16 bit
+         * @return a surfaces object that should render in Imaris (though pixellated)
          * @throws Error an Imaris Error if there was a problem
          */
         public static ISurfacesPrx createFromLabels(ImagePlus impLabel) throws Error {
-            // Check if it has a Time Index Property
-            Object tInd = impLabel.getProperty("Time Index");
-            if (tInd != null) {
-                return createFromLabels(impLabel, (int) tInd);
-            }
-            log.warning("EasyXT cannot find a timepoint associated with this surface mask. Defaulting to Timepoint 0");
-            log.warning("Use Surfaces.createFromLabels(ImagePlus imp, int timepoint) to specify the desired timepoint to insert this surface");
 
-            return createFromLabels(impLabel, 0);
-        }
+            int width = impLabel.getWidth();
+            int height = impLabel.getHeight();
+            int slices = impLabel.getNSlices();
+            int timepoints = impLabel.getNFrames();
 
-        ;
 
-        /**
-         * @param impLabel        the image to get a Surfaces from.
-         *                        A label image, each label will be a surface of the Surfaces object.
-         * @param timepointOffset an index to offset the start of the surface creation.
-         *                        for single timepoint Images, this is effectively the timepoint at which to place the surface
-         * @return the ISurfacesPrx with individual surface for each label value
-         * @throws Error an Imaris Error if there was a problem
-         */
+            ILabelImagePrx labelImage = EasyXT.Utils.getImarisApp().GetFactory().CreateLabelImage();
+            labelImage.Create( width, height, slices, timepoints );
 
-        public static ISurfacesPrx createFromLabels(ImagePlus impLabel, int timepointOffset) throws Error {
+            // Calibrate it to match the image being given
+            Calibration cal = impLabel.getCalibration();
+            labelImage.SetExtendMinX( (float) cal.xOrigin);
```

Comment:
Still the same story: xOrigin is supposed to be in number of pixels. So we do not respect this convention. This is problematic for compatibility with Bdv-Playground, and maybe we don't care. However, I wonder if this is also problematic for cropped images. 

Anyway it's irrelevant for this PR, and since we do not suffer from this yet, let's say that we don't care. I'll open an issue.